tests/test_views.py

```diff
@@ -10,11 +10,12 @@
 import httplib as http
 import math
 import time
+import datetime
```

Comment:
On line 8, we already `import datetime as dt`.

Remove the redundant import and consolidate all usages to something consistent.